components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -369,22 +390,15 @@ export default class AdminSidebar extends React.Component {
                                         </OverlayTrigger>
                                     </div>}
                             </li>
-
+                            {this.renderRootMenu(AdminDefinition.about, 'fa-info', 'admin.sidebar.about', 'ABOUT')}
                             {this.renderRootMenu(AdminDefinition.reporting, 'fa-bar-chart', 'admin.sidebar.reports', 'REPORTING')}
-                            <AdminSidebarCategory
-                                sectionClass='sections--settings'
-                                parentLink='/admin_console'
-                                icon='fa-gear'
-                                title={
-                                    <FormattedMessage
-                                        id='admin.sidebar.settings'
-                                        defaultMessage='SETTINGS'
-                                    />
-                                }
-                            >
-                                {Object.values(AdminDefinition.settings).map(this.renderSettingsMenu)}
-                            </AdminSidebarCategory>
-                            {this.renderRootMenu(AdminDefinition.other, 'fa-wrench', 'admin.sidebar.other', 'OTHER')}
+                            {this.renderRootMenu(AdminDefinition.user_management, 'fa-users', 'admin.sidebar.userManagement', 'USER MANAGEMENT')}
```

Comment:
As part of the renderRootMenu we take a look into all the items and check different things to only show valid entries. One check that was in the renderSettings menu and is not in renderRootMenu was:
```js
 if (!item.schema) {
    return;
}
```
Probably is a good thing to have there.